experimental/CliffordAlgOrd/src/CliffordOrder.jl

```diff
@@ -181,14 +181,50 @@ is_exact_type(::Type{ZZCliffordOrderElem}) = true
 @doc raw"""
     clifford_order(ls::QuadLat) -> CliffordOrder
 
-Return the Clifford order of the even quadratic lattice $ls$.
+Return the Clifford order of the even lattice $ls$.
+
+# Examples
+
+```jldoctest
+julia> K,a = quadratic_field(-5); OK = maximal_order(K);
+
+julia> C = clifford_order(lattice(quadratic_space(K, 2*identity_matrix(K, 2))))
+Clifford order of even lattice over maximal order of imaginary quadratic field defined by x^2 + 5
+with basis [1, sqrt(-5)] with Gram matrix
+  [2   0]
+  [0   2]
+and coefficient ideals of the lattice
+  2-element Vector{AbsSimpleNumFieldOrderFractionalIdeal}:
```

Comment:
same here. at least the `2-element Vector{AbsSimpleNumFieldOrderFractionalIdeal}:` shouldn't be here IMO. The rest also looks ugly, but that could just be due to missing pretty-printing on the Hecke side for fractional ideals.

ping @thofma

Reply:
https://github.com/thofma/Hecke.jl/pull/1840